### PR TITLE
#365 fix location filter

### LINKED ISSUE #365: Countries filter does not work
**Describe the bug**
Searching for countries from the start page does not work.

**To Reproduce**
1. Go to `https://cai-einander-helfen-staging.fbi.h-da.de/`
2. Change to international search
3. Insert one country name and click search button
4. All international results are displayed

**Expected behavior**
Only results from the country you are looking for should be displayed.

**Solution**
The query parameter `location` in the URL is not set.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version 89.0b15

### PR DESCRIPTION
**Pull request by xcaverxiii**
fixed #365 

close #365

### CODE REVIEW

**Review by DianaPankratz — CHANGES_REQUESTED** [review]
The location filter works but now we have again the following problem: If you choose a country/city with enter, the search starts immediately but enter in this case should only set the county/city as the location. Only a secons preess on enter should start the search.

**Comment by xcaverxiii** [comment]
> The location filter works but now we have again the following problem: If you choose a country/city with enter, the search starts immediately but enter in this case should only set the county/city as the location. Only a secons preess on enter should start the search.

now better?

**Comment by DianaPankratz** [comment]
> > The location filter works but now we have again the following problem: If you choose a country/city with enter, the search starts immediately but enter in this case should only set the county/city as the location. Only a secons preess on enter should start the search.
> 
> now better?

Yes this works. 
But I just found another bug. The radius is not working correctly. If I choose a radius except "Überall" I will get results but I also get the message "Zu Ihrer Suchanfrage mit einem Radius von 5km haben wir keine Treffer gefunden. Folgende Ergebnisse werden in einem Umkreis von 25km gefunden."
The search itself is working. A lower radius will give me less results. But the message is not right,

image

**Comment by xcaverxiii** [comment]
> But I just found another bug. The radius is not working correctly. If I choose a radius except "Überall" I will get results but I also get the message "Zu Ihrer Suchanfrage mit einem Radius von 5km haben wir keine Treffer gefunden. Folgende Ergebnisse werden in einem Umkreis von 25km gefunden."
> The search itself is working. A lower radius will give me less results. But the message is not right,

I think this will be fixed after PR #383 is merged automatically. 
This is outside the scope of this issue. Wait for the merge and, if necessary, make a new issue.

File: app/src/components/search/LocationSearchBar.vue [hunk 1]
```diff
@@ -132,6 +134,7 @@ export default Vue.extend({
       // reduce to only one word
       this.searchValue = this.searchValue.split(" ")[0];
       this.$emit("input", this.location);
+      // this.$emit("enter", this.location);
```
Comment (theErikss0n) [inline app/src/components/search/LocationSearchBar.vue:137]: Please remove this line of code if it is not needed anymore.

```suggestion
```

**Review by theErikss0n — APPROVED** [review]
Ca be merged after the single comment is addressed.

**Comment by DianaPankratz** [comment]
I can't reproduce this bug so this is approved.

**Review by DianaPankratz — APPROVED** [review]

**Pull request merged**